manticore/ethereum.py

```diff
@@ -389,17 +389,25 @@ def simplify(x):
             value = arithmetic_simplifier(Operators.CONCAT(size, *map(Operators.ORD, data[offset + padding:offset + padding + byte_size])))
             return simplify(value)
 
-        if ty == u'uint256':
-            return get_uint(256, offset), offset + 32
-        elif ty in (u'bool', u'uint8'):
+        if ty.startswith('uint') and 0 <= int(ty[4:]) <= 256:
+            size = int(ty[4:])
+            if not ((size % 8) == 0):
+                raise EthereumError('Invalid uint size %s' % size)
```

Comment:
actually this will crash since EthereumError is not defined here, we need to import it